tools/shared.py

```diff
@@ -513,7 +513,7 @@ def get_emscripten_version(path):
 # For the Emscripten-specific WASM metadata section, follows semver, changes
 # whenever metadata section changes structure
 # NB: major version 0 implies no compatibility
-(EMSCRIPTEN_METADATA_MAJOR, EMSCRIPTEN_METADATA_MINOR) = (0, 0)
+(EMSCRIPTEN_METADATA_MAJOR, EMSCRIPTEN_METADATA_MINOR) = (0, 1)
```

Comment:
sorry, the right thing is `EMSCRIPTEN_ABI_MINOR`, a few lines down - the metadata structure did not change, the ABI did.

lgtm with that (bump 1 to 2 there)